internal/server/config.go

```diff
@@ -33,6 +33,21 @@ type Jobs struct {
 	ExpiringAccessNotification JobConfig `mapstructure:"expiring_access_notification"`
 }
 
+type DefaultAuth struct {
+	HeaderKey string `mapstructure:"header_key" default:"X-Auth-Email"`
+}
+
+type OIDCAuth struct {
```

Comment:
@swallowstalker would it make sense to move this config next to OIDC auth only? and server config can import it from there.

Reply:
hi @ravisuhag, I've moved the OIDCAuth config to pkg/auth, i think it's more clear that way